diff --git a/Cargo.lock b/Cargo.lock
index 05f9940..0af6345 100644
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -594,39 +594,6 @@ dependencies = [
  "url",
 ]
 
-[[package]]
-name = "gpm"
-version = "0.18.0"
-dependencies = [
- "anyhow",
- "base64",
- "clap",
- "console",
- "crossterm",
- "crypto-hash",
- "dirs",
- "dotenv",
- "err-derive",
- "flate2",
- "git2",
- "gitlfs",
- "indicatif",
- "log",
- "openssl-probe",
- "pest",
- "pest_derive",
- "pretty_env_logger",
- "regex",
- "rpassword",
- "semver",
- "tar",
- "tempfile",
- "termimad",
- "url",
- "vergen",
- "zeroize",
-]
-
 [[package]]
 name = "h2"
 version = "0.3.3"
@@ -978,6 +945,39 @@ dependencies = [
  "winapi",
 ]
 
+[[package]]
+name = "mkpm"
+version = "0.18.0"
+dependencies = [
+ "anyhow",
+ "base64",
+ "clap",
+ "console",
+ "crossterm",
+ "crypto-hash",
+ "dirs",
+ "dotenv",
+ "err-derive",
+ "flate2",
+ "git2",
+ "gitlfs",
+ "indicatif",
+ "log",
+ "openssl-probe",
+ "pest",
+ "pest_derive",
+ "pretty_env_logger",
+ "regex",
+ "rpassword",
+ "semver",
+ "tar",
+ "tempfile",
+ "termimad",
+ "url",
+ "vergen",
+ "zeroize",
+]
+
 [[package]]
 name = "native-tls"
 version = "0.2.7"
diff --git a/Cargo.toml b/Cargo.toml
index 1f33814..cfdcf51 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
-name = "gpm"
+name = "mkpm"
 version = "0.18.0"
-authors = ["Jean-Marc Le Roux <jeanmarc.leroux@aerys.in>"]
+authors = ["Clay Risser <clayrisser@gmail.com>", "Jean-Marc Le Roux <jeanmarc.leroux@aerys.in>"]
 edition = "2018"
 
 [profile.release]
diff --git a/src/gpm/file.rs b/src/gpm/file.rs
index 06323dc..566b0b3 100644
--- a/src/gpm/file.rs
+++ b/src/gpm/file.rs
@@ -9,7 +9,7 @@ use tar::Archive;
 use indicatif::{ProgressBar, ProgressStyle};
 
 pub fn get_or_init_dot_gpm_dir() -> Result<path::PathBuf, io::Error> {
-    let dot_gpm = dirs::home_dir().unwrap().join(".gpm");
+    let dot_gpm = dirs::home_dir().unwrap().join(".mkpm");
 
     if !dot_gpm.exists() {
         return match fs::create_dir_all(&dot_gpm) {
diff --git a/src/main.rs b/src/main.rs
index a9c66e6..7b00b3b 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -28,7 +28,7 @@ fn main() {
 
     pretty_env_logger::init_custom_env("GPM_LOG");
 
-    let matches = App::new("gpm")
+    let matches = App::new("mkpm")
         .about("Git-based package manager.")
         .version(env!("VERGEN_BUILD_SEMVER"))
         .setting(clap::AppSettings::ArgRequiredElseHelp)
